.github/dependabot.yml

```diff
@@ -7,3 +7,10 @@ updates:
     target-branch: "develop"
     labels:
       - "dependencies"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Do we want to update actions so often? We use some third-party actions pinned to commits. When we update them, we have to audit them and verify they still work with our rarely-run release workflows. Ideally, I'd just leave them unless they cause an issue.

Reply:
Yeah not bothered about the cadence just it seems easy to ignore the deprecation warnings like we have.

At some point many of the deprecations would have become fails; I personally like the little and often approach of mending things.

I suspect this will also be silent almost always; we'd have probably only got 4 PRs over the last 2 years maybe?